Build.fsx

```diff
@@ -1,4 +1,4 @@
-﻿#r @"packages/FAKE.4.19.0/tools/FakeLib.dll"
+﻿#r @"packages/FAKE.Core/tools/FakeLib.dll"
```

Comment:
It's hard to get an overview of the changes related to updating FAKE when there are another 150K+ deleted lines. Could you please commit the deletion of the old package separately?